src/clr-angular/layout/vertical-nav/vertical-nav.spec.ts

```diff
@@ -97,6 +98,21 @@ export default function(): void {
 
         expect(vertNavService.collapsed).toBe(false);
       });
+
+      it('sets the aria-expanded attribute to true when the vertical nav is collapsed', () => {
```

Comment:
The text here doesn't describe this test very well.

Reply:
OG contribution, I can update it to make it more descriptive.

Reply:
I updated this description.